extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json

```diff
@@ -51,6 +51,13 @@
             "condition": "if a conditional step has been processed"
           }
         },
+        "xdm:flowControlID": {
+          "title": "Unique identifier for the flow control",
+          "type": "string",
+          "format": "uri",
+          "description":
+            "The identifier of the flow control in a journey verison"
```

Comment:
What is a "journey", seems very product specific.  Also, I assume "verison" is "version", please fix.

Reply:
Journey is product specific in our case ( explained in the description file of this same schema ) , I have fixed the version typo